tests/app/services/file_manager/file_upload/test_upload_client.py

```diff
@@ -330,18 +330,29 @@ def test_output_manifest_success(mocker, tmp_path):
     upload_client = UploadClient('project_code', 'parent_folder_id')
```

Comment:
Could we update the name of this test case to indicate we are also testing the preupload logic in this PR for resumable uploads?